spec/lib/administrate/page/collection_spec.rb

```diff
@@ -0,0 +1,99 @@
+require "active_support/core_ext/module"
+require "administrate/page/collection"
+
+describe Administrate::Page::Collection do
+  # #scope_groups creates the concept of "group of scopes" to manage scopes
+  # always grouped reading Dashboard#collection_scopes (COLLECTION_SCOPES).
+  describe "#scope_groups" do
+    let(:scope) { :mellow }
+    let(:array_of_scopes) { [ scope ] }
+    let(:hash_of_scopes) do
+      {
+        symbol_key: [ :active, :inactive, 'active_since(1992)' ],
+        'string_key': [ 'funny', :sad ]
```

Comment:
Space inside square brackets detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Put a comma after the last item of a multiline hash.